src/gui/graphdesignerpage.h

```diff
@@ -0,0 +1,58 @@
+/**
+*  This file is part of Evoplex.
+*
+*  Evoplex is a multi-agent system for networks.
+*  Copyright (C) 2018 - Marcos Cardinot <marcos@cardinot.net>
```

Comment:
following the license template, here we write the author's name for the file; so, feel free to replace it with your name